lib/ui/page/home/page/chat/widget/video_thumbnail/video_thumbnail.dart

```diff
@@ -131,6 +130,11 @@ class _VideoThumbnailState extends State<VideoThumbnail> {
   /// happened during initialization.
   bool _triedOnError = false;
 
+  /// Indicator whether the video playback was manually stopped.
+  /// If true, the video will not autoplay on mouse hover.
+  /// Otherwise, it will autoplay on mouse hover.
+  bool _hasManuallyStopped = false;
```

Comment:
Also Effective Dart: https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph